CodeGeneration/Sources/generate-swift-syntax/InitSignature+Extensions.swift

```diff
@@ -63,19 +65,25 @@ extension InitSignature {
       )
     }
 
+    func transformParam(_ param: FunctionParameterSyntax) -> FunctionParameterSyntax {
```

Comment:
That feels at odds with how we normally use argument labels because the word `parameter` is redundant information given the type. Should it just be `transform(_:)`?

Reply:
`transform(_:)` seems too ambiguous to me. If you don’t like `transform(parameter:)`, I don’t have super strong opinions on it and we can keep `transformParam(_:)`